Sources/Miscellaneous/Constants.swift

```diff
@@ -10,4 +10,4 @@
 
 import Foundation
 
-let objectNoLongerExists = "Object no longer exists"
+let pubNubSwiftChatSDKVersion = "0.8.2"
```

Comment:
Same question as above.

Reply:
There's `0.8.2-dev` tag in `kmp-chat`. It contains a hotfix for the customer who couldn't resolve dependencies due to large `pubnub-kotlin` submodule size.

A solution made by Wojtek that mitigates this issue was removing `.submodules` file and then creating `0.8.2-dev` tag around it.

Reply:
Thank you for clarification.